src/resources/BreadCrumbResources.ts

```diff
@@ -1,17 +1,14 @@
 import { BreadcrumbIF } from '@bcrs-shared-components/interfaces/'
-import { createPinia, setActivePinia } from 'pinia'
 import { useStore } from '@/store/store'
 
-setActivePinia(createPinia())
-const store = useStore()
-
 /** Returns URL param string with Account ID if present, else empty string. */
 function getParams (): string {
   const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id as string
   return accountId ? `?accountid=${accountId}` : ''
 }
 
 export function getEntityDashboardBreadcrumb (): BreadcrumbIF {
+  const store = useStore()
```

Comment:
Fairly certain we can assume the store has been initialized, because App.vue calls this.